doc: boards: renesas: update board image name with the board name

quytranpzz opened the pull request:
Align Renesas board image name with board name

quytranpzz committed:
doc: boards: renesas: update board image name with the board name

Align Renesas board image name with board name

Signed-off-by: Tran Van Quy <quy.tran.pz@renesas.com>

nordicjm reviewed (dismissed):
Do not change the contents of the files, just rename them. Changing the file contents just bloats the zephyr repo by having to have the OLD AND NEW files contained in git forever

quytranpzz commented:
> Do not change the contents of the files, just rename them. Changing the file contents just bloats the zephyr repo by having to have the OLD AND NEW files contained in git forever

@nordicjm: I corrected it. I added 2 new images for EK-RA2A1 also, cc @soburi
